packages/jest-worker/src/workers/ChildProcessWorker.ts

```diff
@@ -228,6 +234,8 @@ export default class ChildProcessWorker implements WorkerInterface {
       return onProcessEnd(...args);
     };
 
+    this._onCustomMessage = (...arg) => onCustomMessage(...arg);
```

Comment:
```suggestion
    this._onCustomMessage = onCustomMessage;
```

or is it somehow different?